npm error when installing pinia

This issue might be a duplicate of https://github.com/posva/pinia/issues/724

I'm trying to install Pinia via npm but get a

> ERESOLVE unable to resolve dependency tree

error.

### Reproduction

- OS: Win10 and Pop!_OS 21.04
- Node version: v16.13.0
- npm version: 8.1.0
- Vue CLI version: 4.5.15
- Vue version: 3.0.0

The project setup on Win10:

![image](https://user-images.githubusercontent.com/47159597/143847771-b26cc4cb-5d45-4946-acde-ed7e42008513.png)

I created a temporary repository showing the inial commit after creating the project with the Vue CLI

https://github.com/matthiashermsen/temp-pinia

### Steps to reproduce the behavior

1. Clone the repository
2. Run `npm install`
3. Run `npm install pinia`
4. You should get an error

### Expected behavior

It should install the package without errors.

### Actual behavior

This is the terminal error on Win10

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: repo@0.0.0-development
npm ERR! Found: vue@3.2.22
npm ERR! node_modules/vue
npm ERR!   peerOptional vue@"^2 || ^3.0.0-0" from @vue/babel-preset-app@4.5.15
npm ERR!   node_modules/@vue/babel-preset-app
npm ERR!     @vue/babel-preset-app@"^4.5.15" from @vue/cli-plugin-babel@4.5.15
npm ERR!     node_modules/@vue/cli-plugin-babel
npm ERR!       dev @vue/cli-plugin-babel@"~4.5.0" from the root project
npm ERR!   peer vue@"3.2.22" from @vue/server-renderer@3.2.22
npm ERR!   node_modules/@vue/server-renderer
npm ERR!     @vue/server-renderer@"3.2.22" from vue@3.2.22
npm ERR!   5 more (@vue/test-utils, primevue, vue-jest, vue-router, the root project)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! pinia@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vue@3.2.23
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 3" from @vue/composition-api@1.4.0
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.4
npm ERR!     node_modules/pinia
npm ERR!       pinia@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\me\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\me\AppData\Local\npm-cache\_logs\2021-11-26T09_39_36_434Z-debug.log
```

and this is the npm log file

```
0 verbose cli [
0 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
0 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
0 verbose cli   'install',
0 verbose cli   'pinia'
0 verbose cli ]
1 info using npm@8.1.0
2 info using node@v16.13.0
3 timing npm:load:whichnode Completed in 0ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:C:\Program Files\nodejs\node_modules\npm\npmrc Completed in 4ms
6 timing config:load:builtin Completed in 4ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:C:\Users\me\Repositories\repo\.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:C:\Users\me\.npmrc Completed in 1ms
12 timing config:load:user Completed in 1ms
13 timing config:load:file:C:\Users\me\AppData\Roaming\npm\etc\npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:credentials Completed in 1ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 11ms
19 timing npm:load:configload Completed in 11ms
20 timing npm:load:setTitle Completed in 0ms
21 timing npm:load:setupLog Completed in 1ms
22 timing config:load:flatten Completed in 2ms
23 timing npm:load:cleanupLog Completed in 2ms
24 timing npm:load:configScope Completed in 0ms
25 timing npm:load:projectScope Completed in 1ms
26 timing npm:load Completed in 18ms
27 timing arborist:ctor Completed in 1ms
28 timing idealTree:init Completed in 1304ms
29 timing idealTree:userRequests Completed in 4ms
30 silly idealTree buildDeps
31 silly fetch manifest pinia@*
32 timing arborist:ctor Completed in 0ms
33 http fetch GET 200 https://registry.npmjs.org/pinia 1424ms (cache revalidated)
34 silly fetch manifest @vue/composition-api@^1.4.0
35 http fetch GET 200 https://registry.npmjs.org/@vue%2fcomposition-api 87ms (cache revalidated)
36 silly fetch manifest vue@^3.0.0
37 http fetch GET 200 https://registry.npmjs.org/vue 55ms (cache revalidated)
38 silly fetch manifest vue@>= 2.5 < 3
39 silly fetch manifest typescript@~4.1.5
40 http fetch GET 200 https://registry.npmjs.org/typescript 60ms (cache revalidated)
41 silly fetch manifest typescript@^4.4.4
42 timing idealTree Completed in 2998ms
43 timing command:install Completed in 3012ms
44 verbose stack Error: could not resolve
44 verbose stack     at PlaceDep.failPeerConflict (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:544:25)
44 verbose stack     at PlaceDep.place (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:197:21)
44 verbose stack     at new PlaceDep (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\place-dep.js:71:10)
44 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:944:31
44 verbose stack     at Array.map (<anonymous>)
44 verbose stack     at Arborist.[buildDepStep] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:944:8)
44 verbose stack     at async Arborist.buildIdealTree (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:216:7)
44 verbose stack     at async Promise.all (index 1)
44 verbose stack     at async Arborist.reify (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\reify.js:149:5)
44 verbose stack     at async Install.install (C:\Program Files\nodejs\node_modules\npm\lib\install.js:170:5)
45 verbose cwd C:\Users\me\Repositories\repo
46 verbose Windows_NT 10.0.19042
47 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "pinia"
48 verbose node v16.13.0
49 verbose npm  v8.1.0
50 error code ERESOLVE
51 error ERESOLVE could not resolve
52 error
53 error While resolving: [1mrepo[22m@[1m0.0.0-development[22m
53 error Found: [1mvue[22m@[1m3.2.22[22m[2m[22m
53 error [2mnode_modules/vue[22m
53 error   peerOptional [1mvue[22m@"[1m^2 || ^3.0.0-0[22m" from [1m@vue/babel-preset-app[22m@[1m4.5.15[22m[2m[22m
53 error   [2mnode_modules/@vue/babel-preset-app[22m
53 error     [1m@vue/babel-preset-app[22m@"[1m^4.5.15[22m" from [1m@vue/cli-plugin-babel[22m@[1m4.5.15[22m[2m[22m
53 error     [2mnode_modules/@vue/cli-plugin-babel[22m
53 error       [33mdev[39m [1m@vue/cli-plugin-babel[22m@"[1m~4.5.0[22m" from the root project
53 error   [35mpeer[39m [1mvue[22m@"[1m3.2.22[22m" from [1m@vue/server-renderer[22m@[1m3.2.22[22m[2m[22m
53 error   [2mnode_modules/@vue/server-renderer[22m
53 error     [1m@vue/server-renderer[22m@"[1m3.2.22[22m" from [1mvue[22m@[1m3.2.22[22m
53 error   5 more (@vue/test-utils, primevue, vue-jest, vue-router, the root project)
53 error
53 error Could not resolve dependency:
53 error [1mpinia[22m@"[1m*[22m" from the root project
53 error
53 error Conflicting peer dependency: [1mvue[22m@[1m3.2.23[22m[2m[22m
53 error [2mnode_modules/vue[22m
53 error   [35mpeer[39m [1mvue[22m@"[1m>= 2.5 < 3[22m" from [1m@vue/composition-api[22m@[1m1.4.0[22m[2m[22m
53 error   [2mnode_modules/@vue/composition-api[22m
53 error     peerOptional [1m@vue/composition-api[22m@"[1m^1.4.0[22m" from [1mpinia[22m@[1m2.0.4[22m[2m[22m
53 error     [2mnode_modules/pinia[22m
53 error       [1mpinia[22m@"[1m*[22m" from the root project
53 error
53 error Fix the upstream dependency conflict, or retry
53 error this command with --force, or --legacy-peer-deps
53 error to accept an incorrect (and potentially broken) dependency resolution.
53 error
53 error See C:\Users\me\AppData\Local\npm-cache\eresolve-report.txt for a full report.
54 verbose exit 1
```

### Additional information

Based on https://github.com/posva/pinia/issues/724#issuecomment-944964417 I think it points to the correct url ( https://registry.npmjs.org/pinia )

Based on https://github.com/posva/pinia/discussions/727#discussioncomment-1493365 I think there is no `@vue/compositon-api` package installed. I tested `npm remove @vue/composition-api` too.

Same on Mac (MacBook Pro (13-inch, M1, 2020), macOS Monterey)

```
awacode@awacode-MBP13 nuxt3-bivignano % npm install pinia @pinia/nuxt
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: vue@3.2.26
npm ERR! node_modules/vue
npm ERR!   peer vue@"3.2.26" from @nuxt/vite-builder@3.0.0-27338323.1e98259
npm ERR!   node_modules/@nuxt/vite-builder
npm ERR!     @nuxt/vite-builder@"npm:@nuxt/vite-builder-edge@3.0.0-27338323.1e98259" from nuxt3@3.0.0-27338323.1e98259
npm ERR!     node_modules/nuxt3
npm ERR!       dev nuxt3@"^3.0.0-27338323.1e98259" from the root project
npm ERR!   peer vue@"3.2.26" from @nuxt/webpack-builder@3.0.0-27338323.1e98259
npm ERR!   node_modules/@nuxt/webpack-builder
npm ERR!     @nuxt/webpack-builder@"npm:@nuxt/webpack-builder-edge@3.0.0-27338323.1e98259" from nuxt3@3.0.0-27338323.1e98259
npm ERR!     node_modules/nuxt3
npm ERR!       dev nuxt3@"^3.0.0-27338323.1e98259" from the root project
npm ERR!   5 more (@vitejs/plugin-vue, @vue/server-renderer, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! @pinia/nuxt@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vue@2.6.14
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 3" from @vue/composition-api@1.4.3
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.9
npm ERR!     node_modules/pinia
npm ERR!       pinia@"*" from the root project
npm ERR!       1 more (@pinia/nuxt)
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/awacode/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/awacode/.npm/_logs/2022-01-10T00_12_02_442Z-debug.log
```

Current package.json:

{
    "name": "web",
    "version": "0.1.0",
    "private": true,
    "engines": {
        "node": ">=16.0.0 <17.0.0",
        "npm": ">=8.0.0 <9.0.0"
    },
    "engineStrict": true,
    "scripts": {
        "config": "npm exec -- vue config",
        "development": "npm exec -- nodenv --env configs/.env.development --exec npm -- exec -- vue-cli-service serve",
        "dev": "npm run development",
    },
    "dependencies": {
        "@auth0/auth0-spa-js": "1.20.0",
        "core-js": "^3.6.5",
        "logrocket": "2.2.0",
        "rollbar": "2.24.0",
        "vue": "^3.0.0",
        "vue-router": "^4.0.0-0"
    },
    "devDependencies": {
        "@types/auth0": "2.34.13",
        "@types/jest": "24.9.1",
        "@typescript-eslint/eslint-plugin": "^4.18.0",
        "@typescript-eslint/parser": "^4.18.0",
        "@vue/cli": "4.5.15",
        "@vue/cli-plugin-babel": "~4.5.0",
        "@vue/cli-plugin-e2e-cypress": "~4.5.0",
        "@vue/cli-plugin-eslint": "~4.5.0",
        "@vue/cli-plugin-router": "~4.5.0",
        "@vue/cli-plugin-typescript": "~4.5.0",
        "@vue/cli-plugin-unit-jest": "~4.5.0",
        "@vue/cli-plugin-vuex": "~4.5.0",
        "@vue/cli-service": "~4.5.0",
        "@vue/compiler-sfc": "^3.0.0",
        "@vue/eslint-config-prettier": "^6.0.0",
        "@vue/eslint-config-typescript": "^7.0.0",
        "@vue/test-utils": "^2.0.0-0",
        "eslint": "^6.7.2",
        "eslint-plugin-prettier": "^3.3.1",
        "eslint-plugin-vue": "^7.0.0",
        "firebase-tools": "10.1.0",
        "node-env-run": "4.0.2",
        "prettier": "^2.2.1",
        "sass": "^1.26.5",
        "sass-loader": "^8.0.2",
        "typescript": "~4.1.5",
        "vue-jest": "^5.0.0-0"
    }
}
Error when installing:

$ npm install --save-exact pinia
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: pinia@2.0.11
npm ERR! Found: typescript@4.1.6
npm ERR! node_modules/typescript
npm ERR!   peer typescript@">=2" from @vue/cli-plugin-typescript@4.5.15
npm ERR!   node_modules/@vue/cli-plugin-typescript
npm ERR!     dev @vue/cli-plugin-typescript@"~4.5.0" from the root project
npm ERR!   typescript@"~4.1.5" from @vue/cli-ui@4.5.15
npm ERR!   node_modules/@vue/cli-ui
npm ERR!     @vue/cli-ui@"^4.5.15" from @vue/cli@4.5.15
npm ERR!     node_modules/@vue/cli
npm ERR!       dev @vue/cli@"4.5.15" from the root project
npm ERR!   6 more (ts-loader, tslint, tsutils, tsutils, vue-jest, the root project)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional typescript@">=4.4.4" from pinia@2.0.11
npm ERR! node_modules/pinia
npm ERR!   pinia@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: typescript@4.5.5
npm ERR! node_modules/typescript
npm ERR!   peerOptional typescript@">=4.4.4" from pinia@2.0.11
npm ERR!   node_modules/pinia
npm ERR!     pinia@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\karl\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\karl\AppData\Local\npm-cache\_logs\2022-02-19T15_57_58_659Z-debug.log
$ node -v
v16.13.1

$ npm -v
8.1.2

Installed vue via the Vue CLI.

Also Am I supposed to remove Vuex manually?

Same issue here:

```
Attaching to the-hub-nuxt-1
the-hub-nuxt-1  | npm ERR! code ERESOLVE
the-hub-nuxt-1  | npm ERR! ERESOLVE could not resolve
the-hub-nuxt-1  | npm ERR! 
the-hub-nuxt-1  | npm ERR! While resolving: the-hub@1.0.0-alpha.1
the-hub-nuxt-1  | npm ERR! Found: vue@3.2.33
the-hub-nuxt-1  | npm ERR! node_modules/vue
the-hub-nuxt-1  | npm ERR!   peer vue@"^3.2.0" from @headlessui/vue@1.6.1
the-hub-nuxt-1  | npm ERR!   node_modules/@headlessui/vue
the-hub-nuxt-1  | npm ERR!     @headlessui/vue@"^1.6.0" from the root project
the-hub-nuxt-1  | npm ERR!   peer vue@">= 3" from @heroicons/vue@1.0.6
the-hub-nuxt-1  | npm ERR!   node_modules/@heroicons/vue
the-hub-nuxt-1  | npm ERR!     @heroicons/vue@"^1.0.6" from the root project
the-hub-nuxt-1  | npm ERR!   1 more (nuxt)
the-hub-nuxt-1  | npm ERR! 
the-hub-nuxt-1  | npm ERR! Could not resolve dependency:
the-hub-nuxt-1  | npm ERR! pinia@"^2.0.14" from the root project
the-hub-nuxt-1  | npm ERR! 
the-hub-nuxt-1  | npm ERR! Conflicting peer dependency: vue@2.6.14
the-hub-nuxt-1  | npm ERR! node_modules/vue
the-hub-nuxt-1  | npm ERR!   peer vue@">= 2.5 < 3" from @vue/composition-api@1.6.1
the-hub-nuxt-1  | npm ERR!   node_modules/@vue/composition-api
the-hub-nuxt-1  | npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.14
the-hub-nuxt-1  | npm ERR!     node_modules/pinia
the-hub-nuxt-1  | npm ERR!       pinia@"^2.0.14" from the root project
the-hub-nuxt-1  | npm ERR! 
the-hub-nuxt-1  | npm ERR! Fix the upstream dependency conflict, or retry
the-hub-nuxt-1  | npm ERR! this command with --force, or --legacy-peer-deps
the-hub-nuxt-1  | npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
the-hub-nuxt-1  | npm ERR! 
the-hub-nuxt-1  | npm ERR! See /root/.npm/eresolve-report.txt for a full report.
the-hub-nuxt-1  | 
the-hub-nuxt-1  | npm ERR! A complete log of this run can be found in:
the-hub-nuxt-1  | npm ERR!     /root/.npm/_logs/2022-05-17T08_29_57_654Z-debug-0.log
the-hub-nuxt-1 exited with code 1
```

Imo this asks for a package for vue3 only. 

Pinia 2.0 for Vue 2
Pinia 3.0 for Vue 3 

Makes more sense to avoid these issues imo!

```bash
$ npm i @vueuse/core
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: vue@3.2.37
npm ERR! node_modules/vue
npm ERR!   peer vue@">=3.0.3" from @ant-design/icons-vue@6.1.0
npm ERR!   node_modules/@ant-design/icons-vue
npm ERR!     @ant-design/icons-vue@"^6.1.0" from ant-design-vue@3.2.9
npm ERR!     node_modules/ant-design-vue
npm ERR!       ant-design-vue@"^3.2.7" from the root project
npm ERR!   peer vue@">= 3.0.0 < 4" from @fortawesome/vue-fontawesome@3.0.1
npm ERR!   node_modules/@fortawesome/vue-fontawesome
npm ERR!     @fortawesome/vue-fontawesome@"^3.0.1" from the root project
npm ERR!   12 more (@nuxt/vite-builder, @vitejs/plugin-vue, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! @vueuse/core@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vue@2.6.14
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 3" from @vue/composition-api@1.6.3
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.1.0" from @vueuse/core@8.7.5
npm ERR!     node_modules/@vueuse/core
npm ERR!       @vueuse/core@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\admin\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\admin\AppData\Local\npm-cache\_logs\2022-06-27T07_21_21_410Z-debug-0.log
```

same issue here

Same terminal error here:
```
npm install pinia
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: maintenance_frontend@0.0.0
npm ERR! Found: vue@3.1.5
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 3.0.0 < 4" from @fortawesome/vue-fontawesome@3.0.0-5
npm ERR!   node_modules/@fortawesome/vue-fontawesome
npm ERR!     @fortawesome/vue-fontawesome@"^3.0.0-5" from the root project
npm ERR!   peer vue@"3.1.5" from @vue/compiler-sfc@3.1.5
npm ERR!   node_modules/@vue/compiler-sfc
npm ERR!     peer @vue/compiler-sfc@"^3.0.8" from @vitejs/plugin-vue@1.3.0
npm ERR!     node_modules/@vitejs/plugin-vue
npm ERR!       dev @vitejs/plugin-vue@"^1.2.3" from the root project
npm ERR!     dev @vue/compiler-sfc@"^3.1.1" from the root project
npm ERR!   2 more (vue-router, the root project)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! pinia@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vue@3.2.37
npm ERR! node_modules/vue
npm ERR!   peer vue@"^2.6.14 || ^3.2.0" from pinia@2.0.14
npm ERR!   node_modules/pinia
npm ERR!     pinia@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\YFH_T\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\YFH_T\AppData\Local\npm-cache\_logs\2022-06-27T09_32_43_551Z-debug-0.log
```

**Reproduction**
- OS: Win10
- node: v16.15.0
- npm: v8.5.5
- vite: 2.3.8
- vue: 3.1.1

Just to share how I ended up here, if it can help others.

I didn't have any trouble installing pinia and @pinia/nuxt but conflicting dependencies came when I tried to install @vuepic/vue-datepicker

```
While resolving: undefined@undefined
npm ERR! Found: vue@3.2.40
npm ERR! node_modules/vue
npm ERR!   peer vue@">=3.2.0" from @vuepic/vue-datepicker@3.5.2
npm ERR!   node_modules/@vuepic/vue-datepicker
npm ERR!     @vuepic/vue-datepicker@"*" from the root project
npm ERR!   vue@"^3.2.39" from nuxt@3.0.0-rc.11
npm ERR!   node_modules/nuxt
npm ERR!     dev nuxt@"3.0.0-rc.11" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! pinia@"^2.0.23" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vue@2.6.14
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 2.7" from @vue/composition-api@1.7.1
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.4.0" from pinia@2.0.23
npm ERR!     node_modules/pinia
npm ERR!       pinia@"^2.0.23" from the root project
```

I reckon because one is asking for a version after 3.2 and the other one for a version before 2.7
Installing vue in my dependencies fixed the npm i but I would like to understand why pinia is forcing this old version? (If I understood the log correctly)

Thanks for your work!

--- a/packages/docs/ssr/nuxt.md
+++ b/packages/docs/ssr/nuxt.md
@@ -10,6 +10,16 @@ yarn add pinia @pinia/nuxt
 npm install pinia @pinia/nuxt
 ```
 
+:::tip 
+If you're using **npm**, `ERESOLVE unable to resolve dependency tree` error may happen when trying to `npm install` pinia into Nuxt project. This can be fixed by adding the following into your `package.json`:
+
+```js
+"overrides": { 
+  "vue": "latest"
+}
+```
+:::
+
 We supply a _module_ to handle everything for you, you only need to add it to `modules` in your `nuxt.config.js` file:
 
 ```js
@@ -24,16 +34,6 @@ export default defineNuxtConfig({
 ```
 
 And that's it, use your store as usual!
-
-:::tip 
-If you're using **npm**, `ERESOLVE unable to resolve dependency tree` error may happen when trying to `npm install` pinia into Nuxt project. This can be fixed by adding the following into your `package.json`:
-
-```js
-"overrides": { 
-     "vue": "latest"
-}
-```
-:::
 
 ## Using the store outside of `setup()`
 

--- a/packages/docs/ssr/nuxt.md
+++ b/packages/docs/ssr/nuxt.md
@@ -11,7 +11,7 @@ npm install pinia @pinia/nuxt
 ```
 
 :::tip 
-If you're using **npm**, `ERESOLVE unable to resolve dependency tree` error may happen when trying to `npm install` pinia into Nuxt project. This can be fixed by adding the following into your `package.json`:
+If you're using npm, you might encounter an _ERESOLVE unable to resolve dependency tree_ error. In that case, add the following to your `package.json`:
 
 ```js
 "overrides": { 

--- a/packages/docs/ssr/nuxt.md
+++ b/packages/docs/ssr/nuxt.md
@@ -25,21 +25,15 @@ export default defineNuxtConfig({
 
 And that's it, use your store as usual!
 
-### NPM dependency issue with Nuxt 3
-
-As of late December 2022 there is a [known problem](https://github.com/vuejs/pinia/issues/853) that may emerge when running `npm install`. You may end up with following error:
-
-```
-ERESOLVE unable to resolve dependency tree
-```
-
-There is nothing actually wrong with the setup, the problem lies inside npm. However, you can easilly bypass it with adding the following into your `package.json`:
+:::tip 
+If you're using **npm**, `ERESOLVE unable to resolve dependency tree` error may happen when trying to `npm install` pinia into Nuxt project. This can be fixed by adding the following into your `package.json`:
 
 ```js
 "overrides": { 
-    "vue": "latest"
+     "vue": "latest"
 }
 ```
+:::
 
 ## Using the store outside of `setup()`
 

--- a/packages/docs/ssr/nuxt.md
+++ b/packages/docs/ssr/nuxt.md
@@ -25,6 +25,22 @@ export default defineNuxtConfig({
 
 And that's it, use your store as usual!
 
+### NPM dependency issue with Nuxt 3
+
+As of late December 2022 there is a [known problem](https://github.com/vuejs/pinia/issues/853) that may emerge when running `npm install`. You may end up with following error:
+
+```
+ERESOLVE unable to resolve dependency tree
+```
+
+There is nothing actually wrong with the setup, the problem lies inside npm. However, you can easilly bypass it with adding the following into your `package.json`:
+
+```js
+"overrides": { 
+    "vue": "latest"
+}
+```
+
 ## Using the store outside of `setup()`
 
 If you want to use a store outside of `setup()`, remember to pass the `pinia` object to `useStore()`. We added it to [the context](https://nuxtjs.org/docs/2.x/internals-glossary/context) so you have access to it in `asyncData()` and `fetch()`:

